pigallery2.json

```
@@ -0,0 +1,56 @@
+{
+    "PiGallery2": {
+        "containers": {
+            "pigallery2": {
+                "image": "bpatrik/pigallery2",
+                "launch_order": 1,
+                "ports": {
+                    "4001": {
```

According to the instruction I found here, it seems that by default there is a mapping to the port 80 for the container.  So this should be set to 80. I think default of 4001 you set is fine. This was the only way I could make this work.
```suggestion
                    "80": {
```